model/models.js

```diff
@@ -0,0 +1,234 @@
+sap.ui.define([
+	"sap/ui/model/json/JSONModel",
+	"sap/ui/Device"
+], function(JSONModel, Device) {
+	"use strict";
+
+	return {
+
+		createDeviceModel: function() {
+			var oModel = new JSONModel(Device);
+			oModel.setDefaultBindingMode("OneWay");
+			return oModel;
+		},
+		createAppModel: function() {
+
+			var WarehouseData = {
+				PoNumber: "",
+				Uname: "",
+				IssRecLbl: "",
+				Warehouse: "",
+				StorageLoc: "",
+				Plant: "",
+				Print: ""
+			};
+			var WarehouseModel = new JSONModel(WarehouseData);
+			sap.ui.getCore().setModel(WarehouseModel, "Warehouse");

```

Comment:
Why are we setting the model to the Core of UI5? I think will be better to set the model to the views